ramls/isbn.raml

```diff
@@ -0,0 +1,91 @@
+#%RAML 1.0
+title: Inventory API
+version: v1
+protocols: [ HTTP, HTTPS ]
+baseUri: http://localhost
+
+documentation:
+  - title: "ISBN API"
+    content: <b>API for validation and conversion of ISBN-10 and ISBN-13 numbers </b>
+
+types:
+  errors: !include raml-util/schemas/errors.schema
```

Comment:
As far as I can see the logic returns one error maximum. In this case `error.schema` can be used instead

Reply:
Done

Reply:
I might be too late with this comment.

Personally I think that it is better to use a structure which allows for multiple errors. 

It allows for future expansion and is more uniform / consistent with other APIs (the shared RAML defines a [trait](https://github.com/folio-org/raml/blob/raml1.0/traits/validation.raml) for this purpose), and clients can use the same parsing and interpretation logic across APIs.

Reply:
Returned `errors` back.